extras/harmonics_w5/pyproject.toml

```diff
@@ -0,0 +1,102 @@
+[build-system]
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "harmonics_w5"
+version = "0.0.0"
+description = "Weight 5 harmonics"
+readme = "README.md"
+authors = [
+  "A. Barontini <andrea.barontini@mi.infn.it>",
+  "A. Candido <alessandro.candido@mi.infn.it>",
+  "F. Hekhorn <felix.hekhorn@mi.infn.it>",
+  "N. Laurenti <niccolo.laurenti@mi.infn.it>",
+  "G. Magni <gmagni@nikhef.nl>",
+]
+classifiers = [
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Topic :: Scientific/Engineering :: Physics",
+  "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+]
+repository = "https://github.com/N3PDF/eko"
+packages = [
+  { include = "harmonics_w5", from = "src" },
+]
+
+[tool.poetry.dependencies]
+python = "^3.8,<3.11"
+eko ="^0.12.0"
+
+[tool.poetry.group.test]
+optional = true
+
+[tool.poetry.group.test.dependencies]
+pytest = "^7.1.3"
+pytest-cov = "4.0.0"
+pytest-env = "^0.6.2"
+pylint = "^2.12.2"
+
+[tool.poetry.group.dev]
+optional = true
+
+[tool.poetry.group.dev.dependencies]
+pdbpp = "^0.10.3"
+ipython = "^8.0"
+# benchmark
+asv = "^0.4.2"
+virtualenv = "^20.13.2"
+devtools = "^0.10.0"
```

Comment:
I appreciate the idea of keeping a package, and even tests are meaningful (you wrote them, that's useful work, no need to discard). But maybe these are too much...